packages/zoe/test/unitTests/contracts/attestation/expiringNFT/test-addToLiened.js

```diff
@@ -0,0 +1,65 @@
+// @ts-check
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { test } from '@agoric/zoe/tools/prepare-test-env-ava';
+import { makeStore } from '@agoric/store';
+import { AmountMath, makeIssuerKit } from '@agoric/ertp';
+
+import { addToLiened } from '../../../../../src/contracts/attestation/expiring/expiringHelpers';
+import { makeHandle } from '../../../../../src/makeHandle';
+
+test('add for same address', async t => {
+  /** @type {Store<Address,Array<ExpiringAttElem>>} */
+  const store = makeStore('address');
+  const address = 'address1';
+  const handle1 = makeHandle('attestation');
+  const handle2 = makeHandle('attestation');
+
+  const { brand } = makeIssuerKit('external');
+  const amountLiened = AmountMath.make(brand, 0n);
+  const expiration = 0n;
+  const elem1 = {
+    address,
+    handle: handle1,
+    amountLiened,
+    expiration,
+  };
+  const elem2 = { address, handle: handle2, amountLiened, expiration };
+  addToLiened(store, elem1);
+  addToLiened(store, elem2);
+
+  // reuse the same handle. This should not happen, but note that the
+  // store does not do any checking of the handle's uniqueness
+  addToLiened(store, elem1);
```

Comment:
I think it's overkill to try to assert that at the level of the `store` and `addToLiened`, since we already know by construction that the handle is unique because it is only ever created by `makeHandle`. However, I'll add an assertion to `updateLien` so that we can double check that updating a lien to extend the expiration does not introduce a duplicate handle.